dashboard/state_aggregator.py

```diff
@@ -593,7 +593,7 @@ async def list_cluster_events(self, *, option: ListApiOptions) -> ListApiRespons
         all_events = await self._client.get_all_cluster_events()
         for _, events in all_events.items():
             for _, event in events.items():
-                event["time"] = str(datetime.utcfromtimestamp(int(event["timestamp"])))
+                event["time"] = str(datetime.fromtimestamp(int(event["timestamp"])))
```

Comment:
this also changes the schema for cluster event state?

Reply:
Let me double check. I believe it shouldn't. It just uses the different time zone (previously UTC time zone, now local time zone)

Reply:
Before

```
'time': '2023-04-17 14:12:53'
```

After

```
  'time': '2023-04-17 07:12:27'}]
```

so the same schema